assets/src/edit-story/components/panels/link/infoDialog.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useConfig } from '../../../app/config';
+
+const TwoTapFigure = styled.img`
+  min-width: 249px;
+`;
+
+const OneTapFigure = styled.img`
+  min-width: 249px;
+`;
+
+const InfoPaneContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 32px;
+`;
+const InfoPane = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+const InfoText = styled.div`
+  min-width: 180px;
+  margin-left: -48px;
+
+  b {
+    display: block;
+    margin-bottom: 12px;
+  }
+`;
+
+function LinkInfoDialog() {
+  const { pluginDir } = useConfig();
+
+  return (
+    <>
+      <span>
+        {__('Select any element ', 'web-stories')}
+        <b>
+          {__('(excluding a background or fullbleed element)', 'web-stories')}
```

Comment:
Created a PR to improve this, see #707.

We should avoid string concatenation at all costs. This way we can do that.